buildSrc/src/main/java/Dependencies.kt

```diff
@@ -3,7 +3,7 @@ object Versions {
     const val kotlin = "1.5.10"
```

Comment:
iOS requires `1.5.20` at the moment but I haven't been able to use `1.5.20` with compose. Will revisit with `1.5.21`.

Reply:
Yeah, it's a real pain right now that Compose won't work with Kotlin 1.5.20...hopeful that next RC release will.  So, just to clarify, this won't work right now on iOS with Kotlin 1.5.10?

Reply:
No :(. When I tried it said something about not considering `apollo-runtime.klib`. I guess because it's built with `1.5.20`

Reply:
Commit to compile iOS with 1.5.20 like you did for the StarWars app: https://github.com/joreilly/MortyComposeKMM/pull/6/commits/7583493fcd9a65a02de874699c95fc00e9278d9e